src/main/kotlin/com/stefan_grafberger/streamdq/checks/row/ListLengthInRange.kt

```diff
@@ -0,0 +1,55 @@
+package com.stefan_grafberger.streamdq.checks.row
```

Comment:
I think this file is still missing the copyright header, could you add it please?